srcts/src/bindings/input/radio.ts

```diff
@@ -77,7 +83,7 @@ class RadioInputBinding extends InputBinding {
   }
   getState(el: RadioHTMLElement): {
     label: string;
-    value: string[] | number | string;
+    value: ReturnType<RadioInputBinding["getValue"]>;
```

Comment:
That's wild, I've never seen a type system do this

Reply:
So cool, right!?!

So nice to be able to define the types once without having to make a helper type and just let the original definition prevail